tutorials/pulse/4_adding_measurements.ipynb

```diff
@@ -6,14 +6,14 @@
    "source": [
     "# Adding measurements to `Schedule`s\n",
     "\n",
-    "Measurement is clearly a very important part of building a Pulse schedule -- this is required to get the results of our program execution! The powerful low-level control we are granted by Pulse gives us more freedom than `QuantumCircuit`s in specifying how the measurement should be done, enabling you to explore readout error mitigation. This power of course comes with responsibility: we have to understand how measurement works, and accomodate certain hardware constraints.\n",
+    "Measurement is clearly a very important part of building a Pulse schedule -- this is required to get the results of our program execution! The powerful low-level control we are granted by Pulse gives us more freedom than `QuantumCircuit`s in specifying how the measurement should be done, enabling you to explore readout error mitigation. This power of course comes with responsibility: we have to understand how measurement works, and accommodate certain hardware constraints.\n",
```

Comment:
I believe we should change the first this to "it": "building a Pulse schedule -- it is required to get the results", as we refer to the measurement itself.

Reply:
I think the writer should have some freedom to express at her/his own way.. what do you think? or should we treat it according to the grammar as it is open source??

Reply:
As mentioned above - I think even small improvements in text serve everyone, and build on the author's work, rather than overriding their freedom of expression.